backend/api/src/db/category.rs

```diff
@@ -30,6 +33,15 @@ order by index
         children: vec![],
         image_count: it.image_count as u64,
         jig_count: it.jig_count as u64,
+        user_scopes: {
+            let scopes = it
+                .user_scopes
+                .into_iter()
+                .map(|x| UserScope::try_from(x).unwrap())
```

Comment:
`.execpt()` instead of `.unwrap()` would be appropriate for potential debugging purposes